reporter/src/main/kotlin/model/EvaluatedVulnerability.kt

```diff
@@ -0,0 +1,13 @@
+package org.ossreviewtoolkit.reporter.model
+
+import com.fasterxml.jackson.annotation.JsonInclude
+import org.ossreviewtoolkit.model.VulnerabilityReference
+import org.ossreviewtoolkit.model.config.VulnerabilityResolution
+
+data class EvaluatedVulnerability(
+    val id: String,
+    val pkg: EvaluatedPackage,
+    @JsonInclude(JsonInclude.Include.NON_EMPTY)
+    val references: List<VulnerabilityReference>,
+    val resolutions: List<VulnerabilityResolution>
```

Comment:
Why did you decide to use "include non-empty" only for one of the two properties?